src/test/config.spec.js

```diff
@@ -99,4 +99,20 @@ describe("Config", function() {
       expect(config._materialize("rules")).to.deep.equal({ ".read": true });
     });
   });
+
+  describe("#load", function() {
+    it("should try to load default config if no config flag is provided", function() {
+      var config = Config.load({ cwd: _fixtureDir("config-imports") });
+      expect(config._src.hosting).to.equal("hosting.json");
+      expect(config._src.rules).to.equal("rules.json");
+    });
+
+    it("should load config from specified path when config flag is provided", function() {
+      var config = Config.load({
+        cwd: _fixtureDir("valid-config"),
+        configPath: "../valid-config/firebase.json",
```

Comment:
this is the default behavior, isn't it? If you removed all the code that was added, this test would still pass, I think. Should the `cwd` be changed to `invalid-config` and the `configPath` pointed at the correct `firebase.json` to correctly test this behavior (or point to a different file, that isn't `firebase.json`, but would still be valid?)

Reply:
You were correct, the test still passed on master. Fixed the test according to your comment